### PR TITLE
Revert "Don't run automatically on commit (#56)"

### PR DESCRIPTION
**Pull request by jolheiser**
This reverts commit 09004bab21a00641b2f57935ed4f86001c20309e and updates the trigger to use this fork.

### CODE REVIEW

File: src/github.ts [hunk 1]
```diff
@@ -164,4 +164,19 @@ export const addPrComment = async (prNumber: number, comment: string) => {
       json.map((c: { body: string }) => c.body)
     }`,
   );
+
+// trigger GitHub action using workflow_dispatch
+export const triggerBackportAction = async () => {
+  const response = await fetch(
+    `${GITHUB_API}/repos/GiteaBot/gitea-backporter/actions/workflows/backport.yml/dispatches`,
```
Comment (yardenshoham) [inline src/github.ts:171]: Did this line exist before?

**Comment by yardenshoham** [comment]
https://github.com/GiteaBot/gitea-backporter/commit/09004bab21a00641b2f57935ed4f86001c20309e said `yardenshoham` though

File: src/github.ts (hunk 1 above)
Comment (jolheiser) [reply src/github.ts:171]: It did, but was pointing to your repo.

https://github.com/yardenshoham/gitea-backporter/pull/56/files#diff-a248ff5ec76368168c745273fee0601b243545fdca37aa3bec76ef9e9365fdb7L155

Comment (yardenshoham) [reply src/github.ts:171]: So how did it trigger this repo?

**Review by yardenshoham — APPROVED** [review]

**Comment by jolheiser** [comment]
I am unsure why this specifically is failing, do you have any ideas @yardenshoham?

**Comment by yardenshoham** [comment]
Is it a nesting issue?

**Comment by jolheiser** [comment]
Was missing a brace, should be good now.

**Pull request merged**